staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2163,6 +2179,24 @@ type RemoveVolumeOptions struct {
 	DryRun []string `json:"dryRun,omitempty"`
 }
 
+// AddInterfaceOptions is provided when dynamically hot plugging a network interface
+type AddInterfaceOptions struct {
+	// NetworkName indicates the name of the network to which the interface will be connected
```

Comment:
Same, I would use the [original](https://github.com/kubevirt/kubevirt/blob/main/staging/src/kubevirt.io/api/core/v1/schema.go#L1428) description.
"References to a NetworkAttachmentDefinition CRD object. Format:
<networkName>, <namespace>/<networkName>. If namespace is not
specified, VMI namespace is assumed."